downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc

```diff
@@ -28,21 +28,14 @@ To perform a reposync you need a RHEL host that has access to the Internet. Afte
 
 .. If using `--newest-only` the repos downloaded will be ~14GB.
 
-<<<<<<< HEAD
-. If you plan to use Red Hat Single Sign-On (RHSSO) sync these repositories.
-=======
 . If you plan to use Red Hat Single Sign-On (RHSSO) you must also sync these repositories.
->>>>>>> 161c95ba (Fixed formatting and spacing issues (#660))
 
 .. jb-eap-7.3-for-rhel-8-x86_64-rpms
 .. rh-sso-7.4-for-rhel-8-x86_64-rpms
 
 +
 After the reposync is completed your repositories are ready to use with a web server.
 
```

Comment:
```suggestion

```
Suggestion: delete these 2 lines, as the info is replicated in the next step (line 39)

Reply:
Change made; thanks, Aine!